broker/src/test/java/com/xiaoleilu/loServer/handler/HttpResponseHelperTest.java

```diff
@@ -0,0 +1,24 @@
+package com.xiaoleilu.loServer.handler;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class HttpResponseHelperTest {
+    @Test
+    public void getMultipartUploadResponseBody() throws Exception {
+        final String remoteFileName = "foo.txt";
+        final String fileUrl = "http://www.google.com/";
```

Comment:
Never use third party URLs - Use something like `foo.bar`

Reply:
Changed to `file:somefile`.